src/resources/application-dev.properties

```diff
@@ -7,5 +7,5 @@ apache.kafka.address=172.17.0.1:9092
 #Logging configuration
 logging.level.root=info
 logging.level.org.springframework.web=info
-logging.file.path=/var/log
+logging.file.path=~
```

Comment:
Let us also update the instructions on how to change default dictionary of logging file.

Reply:
I updated the instruction on wiki